server/@types/viewModels/index.d.ts

```diff
@@ -414,8 +414,12 @@ declare module 'viewModels' {
     prisonName: string
   }
 
-  export interface PageFlowQueue {
+  export interface PageFlow {
```

Comment:
Renamed, since this is used by both the `PageFlowQueue` and `PageFlowHistory` modules.